src/transforms/push.ts

```diff
@@ -81,7 +84,12 @@ export async function enqueuePush(
 	jiraHost: string,
 	options?: JobOptions
 ) {
-	return queues.push.add(createJobData(payload, jiraHost), options);
+	const queueMesssagePayload = createJobData(payload, jiraHost);
+	if(await booleanFlag(BooleanFlags.SEND_PUSH_TO_SQS, false, jiraHost)) {
+		return sqsQueues.push.sendMessage(queueMesssagePayload);
+	} else {
+		return queues.push.add(createJobData(queueMesssagePayload, jiraHost), options);
```

Comment:
Good lesson to learn here, that even the smallest refactoring can cause problems.

What I wanted to do is just to extract `const queueMesssagePayload = createJobData(payload, jiraHost);` to a variable, however I've put it in a wrong place (as an argument of createJobData function instead of createJobData function invocation itself)

If we deployed it to production it would be broken with feature flag OFF.
cc @mboudreau , @atrigueiro 

Now I'll need to figure out why there were no tests